docs/reference/announcements/870.md

```diff
@@ -0,0 +1,136 @@
+---
+id: announcements-870
+title: "8.7 Announcements"
+description: "Important announcements including deprecation & removal notices"
+---
+
+Important changes and updates for the Camunda 8.7 release that customers should be aware of are summarized below.
+
+## What's coming in Camunda 8.7?
```

Comment:
I suggest we don't call it `What's coming in Camunda 8.7` because this reads like a big overlap to https://docs.camunda.io/docs/next/reference/release-notes/870/, which features `New features in 8.7.0`.

Maybe we just leave this away, since the previous line is already clear enough?
> Important changes and updates for the Camunda 8.7 release that customers should be aware of are summarized below.

Reply:
Agreed, will remove 👍